src/datumaro/version.py

```diff
@@ -1 +1 @@
-__version__ = "1.4.0rc3"
+__version__ = "1.4.0rc4"
```

Comment:
I guess this should be "1.4.0"

Reply:
as a final RC, rc4 will be published to pypi and verified it with Geti & OTX. after then I will be published as 1.4.0.